doc/rados/operations/pools.rst

```diff
@@ -653,227 +644,234 @@ You may set values for the following keys:
+:Description: See min_size_.
 
 :Type: Integer
 :Version: ``0.54`` and above
 
+
 ``pg_num``
 
-:Description: see pg_num_
+:Description: See pg_num_.
 
 :Type: Integer
 
 
 ``pgp_num``
 
-:Description: see pgp_num_
+:Description: See pgp_num_.
 
 :Type: Integer
 :Valid Range: Equal to or less than ``pg_num``.
 
 
 ``crush_rule``
 
-:Description: see crush_rule_
+:Description: See crush_rule_.
 
 
 ``hit_set_type``
 
-:Description: see hit_set_type_
+:Description: See hit_set_type_.
 
 :Type: String
 :Valid Settings: ``bloom``, ``explicit_hash``, ``explicit_object``
 
+
 ``hit_set_count``
 
-:Description: see hit_set_count_
+:Description: See hit_set_count_.
 
 :Type: Integer
 
 
 ``hit_set_period``
 
-:Description: see hit_set_period_
+:Description: See hit_set_period_.
 
 :Type: Integer
 
 
 ``hit_set_fpp``
 
-:Description: see hit_set_fpp_
+:Description: See hit_set_fpp_.
 
 :Type: Double
 
 
 ``cache_target_dirty_ratio``
 
-:Description: see cache_target_dirty_ratio_
+:Description: See cache_target_dirty_ratio_.
 
 :Type: Double
 
 
 ``cache_target_dirty_high_ratio``
 
-:Description: see cache_target_dirty_high_ratio_
+:Description: See cache_target_dirty_high_ratio_.
 
 :Type: Double
 
 
 ``cache_target_full_ratio``
 
-:Description: see cache_target_full_ratio_
+:Description: See cache_target_full_ratio_.
 
 :Type: Double
 
 
 ``target_max_bytes``
 
-:Description: see target_max_bytes_
+:Description: See target_max_bytes_.
 
 :Type: Integer
 
 
 ``target_max_objects``
 
-:Description: see target_max_objects_
+:Description: See target_max_objects_.
 
 :Type: Integer
 
 
 ``cache_min_flush_age``
 
-:Description: see cache_min_flush_age_
+:Description: See cache_min_flush_age_.
 
 :Type: Integer
 
 
 ``cache_min_evict_age``
 
-:Description: see cache_min_evict_age_
+:Description: See cache_min_evict_age_.
 
 :Type: Integer
 
 
 ``fast_read``
 
-:Description: see fast_read_
+:Description: See fast_read_.
 
 :Type: Boolean
 
 
 ``scrub_min_interval``
 
-:Description: see scrub_min_interval_
+:Description: See scrub_min_interval_.
 
 :Type: Double
 
 
 ``scrub_max_interval``
 
-:Description: see scrub_max_interval_
+:Description: See scrub_max_interval_.
 
 :Type: Double
 
 
 ``deep_scrub_interval``
 
-:Description: see deep_scrub_interval_
+:Description: See deep_scrub_interval_.
 
 :Type: Double
 
 
 ``allow_ec_overwrites``
 
-:Description: see allow_ec_overwrites_
+:Description: See allow_ec_overwrites_.
 
 :Type: Boolean
 
 
 ``recovery_priority``
 
-:Description: see recovery_priority_
+:Description: See recovery_priority_.
 
 :Type: Integer
 
 
 ``recovery_op_priority``
 
-:Description: see recovery_op_priority_
+:Description: See recovery_op_priority_.
 
 :Type: Integer
 
 
 Setting the Number of RADOS Object Replicas
 ===========================================
 
-To set the number of object replicas on a replicated pool, execute the following:
+To set the number of object replicas on a replicated pool, run the following
+command:
```

Comment:
@zdover23 - This should probably be _run a command of the following form_.

Reply:
Good catch.